src/main/java/school/lemon/changerequest/java/generics/NumberGenerator/NumberGenerator.java

```diff
@@ -0,0 +1,9 @@
+package school.lemon.changerequest.java.generics.NumberGenerator;
+
+/**
+ * Created by lera on 24.12.16.
+ */
+public abstract class NumberGenerator<T extends Number> {
+
+    public abstract <T extends Number> T[] generateNumbers();
```

Comment:
You do not need to define `T` again, it's already defined in class declaration. So `public abstract T[] generateNumbers();` will be enough.